stdlib/public/Concurrency/Actor.swift

```diff
@@ -86,6 +86,7 @@ public macro isolation<T>() -> T = Builtin.IsolationMacro
 #endif
 
 #if $IsolatedAny
+@_alwaysEmitIntoClient
 @available(SwiftStdlib 5.1, *)
```

Comment:
You can remove this availability since it's AEIC, but it's not hurting anyone either

Reply:
I'm not sure I can, because it's still only usable on runtimes that have concurrent